custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -689,6 +690,25 @@ def _prepare_docker_command(self, options, run_mode, raw_arguments):
 
     def _run_inside_docker(self, options, run_mode, raw_arguments):
         docker_cmd = self._prepare_docker_command(options, run_mode, raw_arguments)
+
+        self._print_verbose("Checking DRUM version in container...")
+        result = subprocess.run(
+            ["docker", "run", "-it", options.docker, "sh", "-c", "drum --version"],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+        )
+        container_drum_version = result.stdout.decode("utf8")
+
+        if container_drum_version != drum_version:
+            print(
+                "WARNING: looks like host DRUM version doesn't match container DRUM version. This can lead to unexpected behavior.\n"
+                "Host DRUM version: {}\n"
+                "Container DRUM version: {}".format(drum_version, result.stdout.decode("utf8"))
+            )
```

Comment:
I would add a small sleep here so the user can see the warning .. maybe 0.5 second sleep in case of version mismatch.

Reply:
done